src/platforms/javascript/common/install/loader.mdx

```diff
@@ -92,15 +92,17 @@ The loader script always includes a call to `Sentry.init` with a default configu
   crossorigin="anonymous"
 ></script>
 <script>
-  Sentry.onLoad(function() { ... });
+  // Check for existence of Sentry in case Ad-blockers block the Sentry Loader Script
+  window.Sentry && Sentry.onLoad(function() { ... });
 </script>
 ```
 
-`Sentry.onLoad` is a function that only the loader provides, and - as the name suggests - it sets a function to be run once the full SDK has been loaded. In that function, you can configure your SDK exactly the way you would were you using the CDN, with one difference: your `Sentry.init` call doesn't need to include your DSN, since it's already been set.
+`Sentry.onLoad` is a function that only the loader provides, and - as the name suggests - it sets a function to be run once the full SDK has been loaded. In that function, you can configure your SDK exactly the way you would were you using the CDN, with one difference: your `Sentry.init` call doesn't need to include your DSN, since it's already been set. Note that we check if `Sentry` exists before we call `Sentry.onLoad` to guard against the case that the loader script failed to load.
```

Comment:
Slight rephrase and I suggest adding some context around ad blocking.


```suggestion
`Sentry.onLoad` is a function that only the loader provides, and - as the name suggests - it sets a function to be run once the full SDK has been loaded. In that function, you can configure your SDK exactly the way you would were you using the CDN, with one difference: your `Sentry.init` call doesn't need to include your DSN, since it's already been set. We recommend checking if `Sentry` exists before calling `Sentry.onLoad` in case the loader script fails to load. One common reason for this is when Sentry is blocked by an ad-blocker. See our docs on [Dealing with Ad-Blockers](https://docs.sentry.io/platforms/javascript/troubleshooting/#dealing-with-ad-blockers) for more information.
```

Reply:
Actually the ad-blocker docs will not apply to this loader usage, as the loader is always loaded from Sentry. So probably this last sentence should not be included...?

Reply:
Oh, sorry about that! I've edited the suggestion.